core/src/main/java/lucee/runtime/db/DCStack.java

```diff
@@ -214,11 +214,9 @@ private boolean isClosedEL(Connection conn) {
 
 	private Boolean isValidEL(Connection conn) {
 		try {
-			// value is in ms but method expect s
-			int ms = datasource.getNetworkTimeout();
-			int s = DEFAULT_TIMEOUT;
-			if (ms > 0) s = (int) Math.ceil(ms / 1000);
-
+			int s = datasource.getNetworkTimeout();
+			//if timeout is unreasonable, use default
+			if (s < 1 || s > 60) s = DEFAULT_TIMEOUT;	
 			return conn.isValid(s) ? Boolean.TRUE : Boolean.FALSE;
```

Comment:
we no longer convert from milliseconds to seconds (`ms / 1000`), is that okay?

Reply:
If you look at the source for [getNetworkTimeout()](https://github.com/lucee/Lucee/blob/5.0/core/src/main/java/lucee/runtime/db/DataSourceSupport.java#L245)  you see it is actually hardcoded to return the constant NETWORK_TIMEOUT_IN_SECONDS ,  which is set to 10.   So the only implementation I could find in the code base was not actually returning milliseconds.  


Also, Math.ceil(10/1000) outputs zero due to using integer division,  so the conversion was never working right anyway.    
![image](https://user-images.githubusercontent.com/14079275/212156526-99cade4c-2440-4895-837d-1c64c05ae958.png)